ui/StatusQ/src/StatusQ/Components/StatusInfoBoxPanel.qml

```diff
@@ -15,6 +15,7 @@ Control {
     property int iconType: StatusInfoBoxPanel.Type.Info
     property alias buttonText: button.text
     property alias buttonVisible: button.visible
+    property alias buttonEnabled: button.enabled
```

Comment:
This is that corner case where using aliases may lead to hard to detect bugs. Described here https://docs.google.com/document/d/1ydhMx1Q6cPJi36W1lRAyBA6eH7Sd7W_jGxpnNpFDes4/edit#heading=h.hrbz4agcn3 (2023-08-19)

Not critical but maybe worth to fix right up front.

Reply:
Ok, I'll update both, `visible` and `enabled` properties. Thanks!